ipynb/tutorial/UseCaseExamples_SchedTuneAnalysis.ipynb

```diff
@@ -568,8 +568,8 @@
    "source": [
     "# Get a list of first 5 \"sched_load_avg_events\" events\n",
     "sched_load_avg_events = !(\\\n",
-    "    grep sched_load_avg_task {trace_file.replace('.dat', '.txt')} | \\\n",
```

Comment:
Good catch... I think that's related to:
   https://github.com/ARM-software/trappy/pull/270
... and specifically this commit:
   https://github.com/ARM-software/trappy/pull/270/commits/159f149e8d8e431d44b875e1785532d06670c993
from mr. @valschneider 

Before that we used to generate and keep a txt file every time Trappy pared a trace file.
We then optimised that by avoiding to keep the `txt` file around since we have both the original file... and also a local cache for already parsed events.